amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-example-large-message.adoc

```diff
@@ -0,0 +1,427 @@
+|===
++
+For example:
++
+image::amazon-sqs-send-message.png[send-message]
+
+=== Add and Configure a Logger Component
+
+Add and configure a *Logger* component to display the response in the Mule console:
+
+. In the *Mule Palette* view, search for *Logger*.
+. Drag the *Logger* component onto the canvas, to the right of the *Send Message* component.
+. Configure the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name for the logger, such as `Log Response`
+|Message |`+++Sent Message: `#[payload]`+++`
+|Level |INFO (Default)
+|===
++
+For example:
++
+image::amazon-sqs-logger.png[logger]
+
+=== Obtain the Number of Messages in the Queue
+
+. In the *Mule Palette* view, search for *Amazon SQS*.
+. Select the *Get approximate number of messages* operation and drag it onto the canvas, to the right of the *Logger* component.
++
+. Configure the *Queue url* field in the properties window, for example:
++
+image::amazon-sqs-get-message-count.png[get-message-count]
+
+=== Add and Configure a Logger Component
+
+Add and configure a *Logger* component to display the number of messages in the Mule console:
+
+. In the *Mule Palette* view, search for *Logger*.
+. Drag the *Logger* component onto the canvas, to the right of the *Get approximate number of messages* component.
+. Configure the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name for the logger, such as `Log Count`
+|Message |`+++Approx. messages in queue: `#[payload]`+++`
+|Level |INFO (Default)
+|===
++
+For example:
++
+image::amazon-sqs-logger2.png[log-message-count]
+
+== Create a Flow to Receive Messages
+
+Finish this example by creating another flow to receive messages and log them before they are deleted from the queue.
+
+. In the Mule Palette view, search for *SQS* and select the *Receive messages* operation:
++
+image::amazon-sqs-select-receive.png[select-receive-messages]
++
+. Drag the *Receive messages* operation onto the canvas.
+. Configure the following fields in the *General* properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name for the connector operation
+|Connector Configuration |Global configuration you created previously `Amazon_SQS_Configuration`. If you want to receive large messages use `Amazon_SQS_Large_Payload_Configuration`.
+|Number of Messages |10
+|Queue url | Amazon SQS queue URL.
+
+If provided, the value of this field takes precedence over the value of the *Default Global Queue URL* field on the *Global Configuration Elements* window.
+|===
++
+For example:
++
+image::amazon-sqs-receive-message.png[receive-message]
++
+. Add a Logger component to display the message in the Mule Console.
++
+. Configure the Logger with these field values:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Name of your choice
+|Message |`+++`#[payload]`+++`
+|Level |INFO (Default)
+|===
+
+== Example Mule Application XML Code
+
```

Comment:
By the time I got here, I had forgotten that there were two flows and one example, so I wonder if we need to reiterate something about the intro text?
```suggestion
After you complete the flow to send a large message to the Amazon queue that references the AWS S3 bucket to use for storing large message payloads, you next create a flow to receive the message and log the message body. 
```